### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by hulien22**
Adding webpack for packaging/building

This allows for us to use third party libraries, have dev builds + better debugging, and no longer have to use the grafana.d.ts file

### CODE REVIEW

**Comment by cognite-cicd** [comment]
[pr-bot]
Run this build with `docker run --rm -p 3000:3000 cognite/grafana-cdp-dev:pr-29`

**Comment by cognite-cicd** [comment]
[pr-bot]
Run this build with `docker run --rm -p 3000:3000 cognite/grafana-cdp-dev:pr-29`

File: tsconfig.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     /* Basic Options */
     "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
-    "module": "system" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
```
Comment (martincognite) [inline tsconfig.json]: Without knowing about everything here, is there a rationale behind changing from `system` to `commonjs`? (I have no idea what `system` means).

**Comment by cognite-cicd** [comment]
[pr-bot]
Run this build with `docker run --rm -p 3000:3000 cognite/grafana-cdp-dev:pr-29`

File: tsconfig.json (hunk 1 above)
Comment (hulien22) [reply tsconfig.json]: `system` refers to systemjs. 

Using `"module": "system"` causes errors with Angular, and according to [this](https://stackoverflow.com/a/35076930), the solution is to transpile to `commonjs` instead.

**Pull request merged**